sale_stock_restocking_fee_invoicing/models/stock_picking.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    is_customer_return = fields.Boolean(compute="_compute_is_customer_return")
```

Comment:
Those fields name can enter in conflict with other modules as too generic.

Either, put them in a separate module, either rename them (with maybe the 'fee' word)